lms/djangoapps/teams/static/teams/js/collections/team_membership.js

```diff
@@ -1,30 +0,0 @@
-;(function (define) {
```

Comment:
That is so bizarre! The file has a long edit history. Anyway, I agree that it isn't being used.